src/darwin/Framework/CHIP/MTRAttestationInfo.h

```diff
@@ -23,6 +23,36 @@ NS_ASSUME_NONNULL_BEGIN
  * Represents information relating to product attestation.
  *
  */
+@interface MTRAttestationInfo : NSObject
+
+@property (nonatomic, copy, readonly) NSData * challenge;
+
+@property (nonatomic, copy, readonly) NSData * nonce;
+
+@property (nonatomic, copy, readonly) NSData * elements;
+
+@property (nonatomic, copy, readonly) NSData * elementsSignature;
+
+@property (nonatomic, copy, readonly) NSData * dac;
+
+@property (nonatomic, copy, readonly) NSData * pai;
```

Comment:
Seeing how we are expanding “NOC”, I’m wondering if we should take this opportunity to also change “dac” and “pai” into longer names?

```suggestion
@property (nonatomic, copy, readonly) NSData * deviceAttestationCert;

@property (nonatomic, copy, readonly) NSData * productAttestationIntermediateCert;
```

Reply:
Done, good catch.